pkg/hubble/metrics/metrics.go

```diff
@@ -64,6 +86,18 @@ func initMetrics(address string, enabled api.Map, grpcMetrics *grpc_prometheus.S
 		errChan <- srv.ListenAndServe()
 	}()
 
+	go func() {
```

Comment:
There is nothing ensuring this go routine gets cleaned up, we should be waiting on it somewhere.

Reply:
I changed the implementation to workqueue and right now goroutine will exit after calling `queue.Shutdown()` but I'm not sure where to put it. A similar goroutine for webserver with metrics seems to not have anything that could stop it. Can you point me where to implement this "shutdown hook"?

Reply:
Good point, I didn't notice that. I guess we don't do properly handle that either. It would require refactoring most of the code in this package to work on a struct with methods. We can probably we can fix this later, I think it's a bit unrelated to this PR.